haystack/nodes/other/shaper.py

```diff
@@ -397,6 +648,8 @@ def run(  # type: ignore
                 ", ".join([f"{key}={value}" for key, value in input_values.items()]),
             )
             output_values = self.function(**input_values)
+            if not isinstance(output_values, tuple):
```

Comment:
Enable single value as return value for shaper functions to simplify usage of shaper functions in PromptTemplate